app/assets/stylesheets/application.scss

```diff
@@ -1,46 +1,67 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 
```

Comment:
having all of your CSS in one gigantic file is going to become really hard to deal with.